webpack.config.js

```diff
@@ -1,48 +1,43 @@
-'use strict';
-
 var path = require('path');
-var args = require('minimist')(process.argv.slice(2));
-
-// List of allowed environments
-var allowedEnvs = ['dev', 'dist', 'test'];
 
-// Set the correct environment
-var env;
-if(args._.length > 0 && args._.indexOf('start') !== -1) {
-  env = 'test';
-} else if (args.env) {
-  env = args.env;
-} else {
-  env = 'dev';
-}
-process.env.REACT_WEBPACK_ENV = env;
+var srcPath = path.join(__dirname, 'src');
 
-// Get available configurations
-var configs = {
-  base: require(path.join(__dirname, 'cfg/base')),
-  dev: require(path.join(__dirname, 'cfg/dev')),
-  dist: require(path.join(__dirname, 'cfg/dist')),
-  test: require(path.join(__dirname, 'cfg/test'))
+module.exports = {
+    entry: [
+      'webpack/hot/dev-server',
+      'webpack-dev-server/client?http://localhost:8080',
+      path.resolve(__dirname, 'src/components/app/app.js')
+    ],
+    resolve: {
+      extensions: ['', '.js', '.jsx']
+    },
+    output: {
+        path: path.resolve(__dirname, 'build'),
+        filename: 'bundle.js',
+    },
```

Comment:
we don't should name the final js file like...katanga.js or something like that
?